### ISSUE TITLE
AWS S3 Bucket where mainnet files live according to docs does not exist

### ISSUE THREAD

**Issue by vastus** [0xPolygonHermez/zkevm-node#2693]
The [Download/Extract Mainnet Files](https://wiki.polygon.technology/docs/zkevm/step2-fullzkevm/#downloadextract-mainnet-files) section has a URL where the mainnet files can be downloaded from but the bucket doesn't exist.

Is still this relevant? Is there an updated bucket URL?

```
$ curl https://de012a78750e59b808d922b39535e862.s3.eu-west-1.amazonaws.com/v1.1.0-rc.1-fork.4.tgz
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchBucket</Code><Message>The specified bucket does not exist</Message><BucketName>de012a78750e59b808d922b39535e862</BucketName><RequestId>9B3H290PZ1E0HPD3</RequestId><HostId>e6wZ39nn39zaqwPESrHh8Gah5NwcLYKY7p2YhC8hhQ+k1ZJiYZ3zatKTt+BPDfy8TOsh/+yY2yuYYYwOFX01Ig==</HostId></Error>
```

**Comment by vtleonardo** [comment]
Yes, there is: "https://storage.googleapis.com/zkevm/zkproverc/v1.1.0-rc.1-fork.4.tgz", I found this buried in the ZKEVM node repo, all development on that repo is happening in the "develop" branch. I believe the team should remove the "main" branch, or at least update the docs on the main. This bucket being offline and the old version being referred to in the docs costed me days. 

OBS: I recommend downloading the fork.5 (v2.0.0-RC4-fork.5.tgz) (if you are using the polygonCDK) or the fork.6 if you are running the Polygon ZKEVM. The "develop" branch has better read-me and a support script to download the files. 

https://github.com/0xPolygonHermez/zkevm-prover/blob/develop/tools/download_archive.sh
